config/game.cfg

```diff
@@ -1,5 +1,7 @@
 // Game specific config stuff goes here
 
+exec "config/lang_en.cfg" //language strings
```

Comment:
You should not have to execute the language string file here, it's already executed in ui.cfg.